project/MimaExcludes.scala

```diff
@@ -181,6 +181,9 @@ object MimaExcludes {
             ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.RealClock"),
             ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.Clock"),
             ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.TestClock")
+          ) ++ Seq(
```

Comment:
OK, right, this is what @ankurdave was saying was an acceptable API change since this isn't meant to be extended anyway and can't technically be extended because a class the API uses isn't public. I'd still rather let him or @jegonzal push the final button as it's touching the API of GraphX.